BarrysPeanutsAkkaCascading/src/main/java/barryspeanuts/actor/CheckOutActor.java

```diff
@@ -0,0 +1,71 @@
+package barryspeanuts.actor;
+
+import akka.actor.typed.ActorSystem;
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.AbstractBehavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.actor.typed.javadsl.Receive;
+import barryspeanuts.msg.CreditCard;
+import barryspeanuts.msg.Customer;
+import barryspeanuts.msg.PurchaseItem;
+import java.util.ArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CheckOutActor extends AbstractBehavior<Object> {
+  Logger logger = LoggerFactory.getLogger(ShoppingCartActor.class);
```

Comment:
```suggestion
  private static final Logger logger = LoggerFactory.getLogger(CheckOutActor.class);
```

(and fix on all other classes)

Reply:
The cascading code has been removed from the project, @cretz .

Reply:
:+1: This comment was from weeks ago. You can mark these issues as resolved if they are resolved.